src/Microsoft.AspNet.Testing/xunit/OperatingSystems.cs

```diff
@@ -13,7 +13,6 @@ public enum OperatingSystems
         Win2008R2 = 1 << 1,
         Linux = 1 << 2,
         MacOSX = 1 << 3,
-
-        Win7And2008R2 = Win7 | Win2008R2,
```

Comment:
@victorhurdugaci do we need this? we could do this on the test itself ...doesn't add much value i think...i will update other repos which uses this